lib/services/northBound/contextServer.js

```diff
@@ -1561,121 +109,26 @@ function updateErrorHandlingNgsiLD(error, req, res, next) {
  * @param {Object} router      Express request router object.
  */
 function loadContextRoutes(router) {
-    //TODO: remove '//' paths when the appropriate patch comes to Orion
-    var updateMiddlewaresNgsi1 = [
-            middlewares.ensureType,
-            middlewares.validateJson(updateContextTemplateNgsi1),
-            handleUpdateNgsi1,
-            updateErrorHandlingNgsi1
-        ],
-        updateMiddlewaresNgsi2 = [
-            middlewares.ensureType,
-            middlewares.validateJson(updateContextTemplateNgsi2),
-            handleUpdateNgsi2,
-            updateErrorHandlingNgsi2
-        ],
-        updateMiddlewaresNgsiLD = [
-            middlewares.ensureType,
-            middlewares.validateJson(updateContextTemplateNgsiLD),
-            handleUpdateNgsiLD,
-            updateErrorHandlingNgsiLD
-        ],
-        queryMiddlewaresNgsi1 = [
-            middlewares.ensureType,
-            middlewares.validateJson(queryContextTemplate),
-            handleQueryNgsi1,
-            queryErrorHandlingNgsi1
-        ],
-        queryMiddlewaresNgsi2 = [
-            handleQueryNgsi2,
-            queryErrorHandlingNgsi2
-        ],
-        queryMiddlewaresNgsiLD = [
-            handleQueryNgsiLD,
-            queryErrorHandlingNgsiLD
-        ],
-        updatePathsNgsi1 = [
-            '/v1/updateContext',
-            '/NGSI10/updateContext',
-            '//updateContext'
-        ],
-        updatePathsNgsi2 = [
-            '/v2/op/update',
-            '//op/update'
-        ],
-        updatePathsNgsiLD = [
-            '/ngsi-ld/v1/entities/:id/attrs/:attr'
-        ],
-        queryPathsNgsi1 = [
-            '/v1/queryContext',
-            '/NGSI10/queryContext',
-            '//queryContext'
-        ],
-        queryPathsNgsi2 = [
-            '/v2/op/query',
-            '//op/query'
-        ],
-        queryPathsNgsiLD = [
-            '/ngsi-ld/v1/entities/:id'
-        ];
```

Comment:
Where this routes configuration has been moved now? I have tried to find the new place, but I'm being unable...

Reply:
The routes are defined in `contextSever-NGSI-LD.js` and the equivalent v1 and v2 files.  Line 41,42 or thereabouts.

Reply:
NTC